### DISCUSSION TITLE
FGRM222 issue on multilevel switch

### DISCUSSION THREAD

**Discussion by Patator33** [zwave-js/zwave-js#6976] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Since 12.12.1, the FGRM222 (v22.22) have no multilevel switch. So i can just open and close with a binary switch.
When i use the wall switch manually, a multlilevel Switch v0 appear with just a current value.
With a FGR222, i have a multilevel switch v3 with possible action like target value.

What did you expect to happen instead?
have a complete multilevel swtich with positionning

re-interview node wiil do nothing but the multilevel swtch v0 desappear.

Anything else we should know?

### Software versions

zwave-js-ui: 9.14.4
zwave-js: 12.12.1
home id: 3527579196
home hex: 0xd242963c
install on LXC promox with tteck script

### Device information

Manufacturer: Fibaro
Model name: FGRM 222
Node ID: ...

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-07-04.log](https://github.com/user-attachments/files/16097227/zwavejs_2024-07-04.log)

**Comment by AlCalzone** [comment]
The device claims not to support Multilevel Switch:
```
2024-07-04T10:17:46.594Z CNTRLR « [Node 023] node info received
                                  supported CCs:
                                  · Manufacturer Specific
                                  · Version
                                  · Configuration
                                  · Association
                                  · Binary Switch
                                  · Powerlevel
                                  · Meter
                                  · Multilevel Sensor
                                  · Firmware Update Meta Data
                                  · Binary Switch
                                  · Manufacturer Proprietary
                                  · Protection
```

I wonder if you have to change the operating mode and then exclude and re-include them for Multilevel Switch to be advertised.
I did have those devices a couple of years ago and I'm pretty sure Multilevel Switch was in the list.

**Comment by wigster** [comment]
I recently re-interviewed all my nodes and all the fgrm-222 lost the multilevel switches. They had been working fine up to this point with the multilevel switches since I added them to ZwaveJS a couple of year ago. All I get in HA is a switch entity which closes and opens but venetian tilt or position support.

**Comment by Patator33** [comment]
I have test to exclude and re-include and no Multilevel switch advertised.
I'am sure that the multilevel switch is normaly in the list.
Two weeks ago, i used this module with Zwavejsui with 2 version late and it was working fine ! (with multilevel switch so)

**Comment by AlCalzone** [comment, marked as answer]
Escalated to an issue: https://github.com/zwave-js/node-zwave-js/issues/6985
Please follow-up there.
